src/client.ts

```diff
@@ -157,9 +157,10 @@ export class Client {
             opts = copy(options)
             opts.agent = options.agent
         }
-        opts.addressPrefix = 'STX'
-        opts.chainId = '79276aea5d4877d9a25892eaa01b0adf019d3e5cb12a97478df3298ccdd01673'
-        return new Client('https://testnet.steem.vc', opts)
+
+        opts.addressPrefix = 'TST'
```

Comment:
These assignments could be an issue. `addressPrefix` & `chainId` are not guaranteed on any testnet, including `testnet.steemitdev.com`; in fact there are explicit stories [and direction on the dev portal](https://developers.steem.io/quickstart/#quickstart-testnet) about the chain id changing to something derived from the current git-hash. 

> Steemit maintains a live testnet to which you can connect. In the near future, we expect the chain id to update with every code change.

We need to have these update only-if-not-set.

Reply:
Looking for smallest set of changes in this PR, let's revisit this later. Still need to figure out how we'll handle it in the other clients too.

Reply:
> Still need to figure out how we'll handle it in the other clients too.
I agree, but that's not relevant to this client. 

Added issue #5 to track our needs, and approving to keep the flow going.